hokusai/staging.yml

```diff
@@ -48,12 +91,23 @@ metadata:
     component: web
   name: aprd-web
   namespace: default
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:iam::585031190124:server-certificate/2018-01-17_artsy-net-wildcard"
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
```

Comment:
Remove `service.beta.kubernetes.io/aws-load-balancer-backend-protocol` and add... `service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"`